config/ocm/crd/manifestwork.yaml

```diff
@@ -0,0 +1,558 @@
+                                interested field.
+                              items:
+                                properties:
+                                  fieldValue:
+                                    description: Value is the value of the status
+                                      field. The value of the status field can only
+                                      be integer, string or boolean.
+                                    properties:
+                                      boolean:
+                                        description: Boolean is bool value when type
+                                          is boolean.
+                                        type: boolean
+                                      integer:
+                                        description: Integer is the integer value
+                                          when type is integer.
+                                        format: int64
+                                        type: integer
+                                      string:
+                                        description: String is the string value when
+                                          when type is string.
+                                        type: string
+                                      type:
+                                        description: Type represents the type of the
+                                          value, it can be integer, string or boolean.
+                                        enum:
+                                        - Integer
+                                        - String
+                                        - Boolean
+                                        type: string
+                                    required:
+                                    - type
+                                    type: object
+                                  name:
+                                    description: Name represents the alias name for
+                                      this field. It is the same as what is specified
+                                      in StatuFeedbackRule in the spec.
+                                    type: string
+                                required:
+                                - fieldValue
+                                - name
+                                type: object
+                              type: array
+                              x-kubernetes-list-map-keys:
+                              - name
+                              x-kubernetes-list-type: map
+                          type: object
+                      type: object
+                    type: array
+                type: object
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
```

Comment:
What's the reason for the duplicate crds ? https://github.com/Kuadrant/multicluster-gateway-controller/blob/15f25d153ac2f5826552e4c78ef4b1b3b130aecf/config/ocm/crd/manifestwork.crd.yaml, or are they different somehow?

Reply:
Thats a mistake, I've removed that and others that were duplicates i think were created from rebasing the ocm into main